css/style.css

```diff
@@ -0,0 +1,128 @@
+html,
+body {
+  min-width: 290px;
+  color: #4d4e53;
+  background-color: #fff;
+  font-family: "Open Sans", Arial, sans-serif;
+  line-height: 1.5;
+}
+
+#navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0;
+  left: 0;
```

Comment:
Left and top measure was with 0px and it is changed to 0 as it doesn't have to be assigned a measurement (unit)

Reply:
kindly it is better to always start with courteous words like please or kindly. know that it is not an order of that you know rather it is an orientation.